content/source/docs/enterprise/install/installer.html.md

```diff
@@ -52,13 +52,41 @@ This change updates the proxy settings for the Terraform Enterprise application
 `NOTE: Please take precautions on application outage when applying configuration change, i.e. wait for all runs to finish, prevent new runs to trigger`
 4. Restart the Replicated services following [the instructions for your distribution](https://help.replicated.com/docs/native/customer-installations/installing-via-script/#restarting-replicated).
 
-## Trusting SSL/TLS Certificates
+## TLS Configuration
 
-There are two primary areas for SSL configuration.
+There are two sections for TLS configuration; the TLS Key & Cert section and the SSL/TLS Configuration section.
```

Comment:
```suggestion
There are two sections for TLS configuration; the "TLS Key & Cert" section and the "SSL/TLS Configuration" section.
```

Quote marks are helpful for identifying UI text; I'll have a couple more of these tweaks below.